src/main/java/org/dsa/iot/knx/DevicePoint.java

```
@@ -33,7 +36,17 @@ public PointType getType() {
 
 	@Override
 	public GroupAddress getGroupAddress() {
-		return this.groupAddress;
+		String addressStr = node.getAttribute(ATTR_INDIVIDUAL_ADDRESS).getString();
+		IndividualAddress address = null;
+		try {
+			address = new IndividualAddress(addressStr);
+		} catch (KNXFormatException e) {
+			e.printStackTrace();
```

same here. I'd rather see it fail fast